File: get-feature-13.py
import sqlite3
import json

db = sqlite3.connect('./features.db')
cursor = db.cursor()

cursor.execute('SELECT * FROM features WHERE id = ?', (13,))
columns = [description[0] for description in cursor.description]
row = cursor.fetchone()

if row:
    result = dict(zip(columns, row))
    print(json.dumps(result, indent=2))
else:
    print('Feature #13 not found')

db.close()

File: get-feature-13.js
const sqlite3 = require('sqlite3').verbose();

const db = new sqlite3.Database('./features.db', (err) => {
  if (err) {
    console.error('Error opening database:', err.message);
    process.exit(1);
  }
});

db.get('SELECT * FROM features WHERE id = ?', [13], (err, row) => {
  if (err) {
    console.error('Error:', err.message);
    process.exit(1);
  }
  console.log(JSON.stringify(row, null, 2));
  db.close();
});
